.gitattributes

```diff
@@ -0,0 +1,3 @@
+# Ensure all Java files use LF.
+*.java eol=lf
+
```

Comment:
I think this is a extra new line. Can you check?

Reply:
Fixed in [f569dee](https://github.com/ballerina-platform/static-code-analysis-tool/pull/2/commits/f569deea8e438e558f4505d64002edaa768de6f0)